### PR TITLE
chore(disable-anniversary-hat): :recycle: disable anniversary theme based on env var

### PR DESCRIPTION
**Pull request by tukantje**
# Summary
Disables anniversary themes based on env var, default true.

# To Test
1. Go to cow swap
2. You should not see birthday hat

### CODE REVIEW

**Review by alfetopito — COMMENTED** [review]
Anything against simply reverting the commit which added the hat in the first place?

File: .env [hunk 1]
```diff
@@ -92,3 +92,6 @@ REACT_APP_FEEDBACK_ENABLED_DEV=false
 
 # Launch darkly
 REACT_APP_LAUNCH_DARKLY_KEY=643681f370ea6d135128b2ce
+
+# Toggle anniversary themes
+REACT_APP_IS_ANNIVERSARY=false
```
[the remark this answers is not in the document]
Comment (tukantje) [reply .env:97]: I avoided that specifically because I thought it would add to our usage right?

**Comment by tukantje** [comment]
> I don't see the hat, but I can see the confetti after the 1st order placement image

Looking at it - that's odd indeed

**Comment by tukantje** [comment]
> Anything against simply reverting the commit which added the hat in the first place?

We could - but I presume we'll need to do this next year as well. 🤷‍♂️

**Comment by tukantje** [comment]
> > I don't see the hat, but I can see the confetti after the 1st order placement image
> 
> Looking at it - that's odd indeed

@elena-zh apparently we are not using TransactionMod here, updated. Now it should not show.

File: .env (hunk 1 above)
[the remark this answers is not in the document]
Comment (tukantje) [reply .env:97]: cc/ @alfetopito

[the remark this answers is not in the document]
Comment (alfetopito) [reply .env:97]: Not sure it's a separated request (I would expect them to return all flags in a single call), but doesn't matter at this point, as per the discussion earlier this morning.

**Comment by alfetopito** [comment]
> > Anything against simply reverting the commit which added the hat in the first place?
> 
> We could - but I presume we'll need to do this next year as well. 🤷‍♂️

That's next year problem =P
Seriously though, no guarantee the app will look the same 1y from now.
We could come back and re-apply the PR if we really want to re-use this IMO

**Comment by tukantje** [comment]
> > > Anything against simply reverting the commit which added the hat in the first place?
> > 
> > 
> > We could - but I presume we'll need to do this next year as well. 🤷‍♂️
> 
> That's next year's problem =P Seriously though, no guarantee the app will look the same 1y from now. We could come back and re-apply the PR if we really want to re-use this IMO

Alright - replaced commits with a revert commit.

**Review by alfetopito — APPROVED** [review]

**Review by shoom3301 — APPROVED** [review]

**Review by anxolin — COMMENTED** [review]
All good, just the title of the PR is wrong

Why env??

**Pull request merged**